core/src/main/scala/org/bitcoins/core/serializers/PicklerKeys.scala

```diff
@@ -48,6 +48,7 @@ object PicklerKeys {
   final val outcomesKey: String = "outcomes"
 
   final val torStartedKey: String = "torStarted"
+  final val syncKey: String = "sync"
```

Comment:
I would name this to `syncing` rather than `sync`. `sync` is ambigous. If `sync=true` does that mean we are syncing with the network, or we are `synced` with the network? 

If we say `syncing=true` it synchronization is in progress IMO.

Reply:
`syncing` or `synced` sounds good.